cudax/include/cuda/experimental/__stf/graph/graph_ctx.cuh

```diff
@@ -517,6 +517,23 @@ public:
     cudaGraphDebugDotPrint(get_graph(), filename.c_str(), flags);
   }
 
+  template <typename T>
+  auto transfer_host(cuda::experimental::stf::logical_data<T>& ldata)
```

Comment:
As discussed, this would be best renamed as `wait` or `host_wait`. There should be a clear messaging that this is a blocking function (unless most of cudastf).

Reply:
I disagree wait suggests the main goal is to synchronize with the data, while the goal is to get its content, synchronization is a necessary byproduct 

we could also have a wait method on logical data we have missed that for ages

Reply:
I'll rename it to wait ... to recall the similarities with std::future